### ISSUE TITLE
Better UI for tablets

### ISSUE THREAD

**Issue by xmha97** [z-huang/music#1083]
### Checklist

- [X] I've checked that there is no other issue with this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

You should move the Bottom navigation to the left side like the Paisa app on large screens.

# Paisa
https://github.com/RetroMusicPlayer/Paisa/releases
## Phone
![image](https://github.com/z-huang/InnerTune/assets/23485114/93bf7075-f399-4bb1-a4a2-b3cc5c4401b9)
## Tablet
![image](https://github.com/z-huang/InnerTune/assets/23485114/1efde2d2-7fa1-474a-a467-42366b2a7510)
# InnerTube
## Phone
![image](https://github.com/z-huang/InnerTune/assets/23485114/b823279b-ddbf-4e40-b5a6-e8d083261775)
## Tablet
![image](https://github.com/z-huang/InnerTune/assets/23485114/a9a866ec-f6a2-4cd9-aa3f-7114e8e22613)

### Why do you want this feature?

Better access to tabs

### Additional information

I using InnerTune on Windows 11 (using WSA)

**Comment by CodePolygon** [comment]
Yes, Definitely Please add a support for table or bigscreen, I am using it on linux through waydroid, It works really great.
